packages/desktop-client/src/components/budget/rollover/budgetsummary/ToBudget.tsx

```diff
@@ -46,7 +46,7 @@ export function ToBudget({
 }: ToBudgetProps) {
   const [menuOpen, setMenuOpen] = useState(null);
   const sheetName = useSheetName(rolloverBudget.toBudget);
-  const sheetValue = useSheetValue({
+  const sheetValue = useSheetValue<string>({
     name: rolloverBudget.toBudget,
     value: 0,
   });
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏this is what I was afraid of. We manually type this as a `string`, but the value here is clearly a `number`.

Reply:
When `sheetValue` is used below, it's passed into `parseInt`, so I assumed it would be a string. Maybe I'm misunderstanding how `useSheetValue` works?

Reply:
For now - ignore how `sheetValue` is used. Focus on the three highlighted lines.

We provide `value: 0` in there as the default value. And then we also manually type the return type as `string`. So which type will be returned? According to the typings: `string`. But in reality it could as well be `number`.

Sample: https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABCAzgUwMoAs1qgNQEMAbENAHgBUA+ACgDcSyAuZMAazDgHcwBKVpUQBvAFCIJiAE54QUpI1JpEhFIkoBuUQF9RoiAhRREKHHiJLEAXmTpsuAkwpGpMMAHM6ABj5bRAen9EGDUAA1MHCzJQlUQwEABbACM0KUQ4NMITKFcPAH4gA